doc/howto/snap.md

```diff
@@ -0,0 +1,166 @@
+This method can cause a problem for cluster updates if some cluster members are refreshed to a version that is not available to other cluster members yet.
+
+To avoid this problem, use the `--cohort="+"` flag when refreshing your snaps:
+
+    sudo snap refresh lxd --cohort="+"
+
+This flag ensures that all machines in a cluster see the same snap revision and are therefore not affected by a progressive rollout.
+
+### Use a Snap Store Proxy
+
+If you manage a large LXD cluster and you need absolute control over when updates are applied, consider installing a Snap Store Proxy.
+
+The Snap Store Proxy is a separate application that sits between the snap client command on your machines and the snap store.
+You can configure the Snap Store Proxy to make only specific snap revisions available for installation.
+
+See the [Snap Store Proxy documentation](https://docs.ubuntu.com/snap-store-proxy/) for information about how to install and register the Snap Store Proxy.
+
+After setting it up, configure the snap clients on all cluster members to use the proxy.
+See [Configuring snap devices](https://docs.ubuntu.com/snap-store-proxy/en/devices) for instructions.
+
+You can then configure the Snap Store Proxy to override the revision for the LXD snap:
+
+    sudo snap-proxy override lxd <channel>=<revision>
+
+For example:
+
+    sudo snap-proxy override lxd stable=25846
+
+## Configure the snap
+
+The LXD snap has several configuration options that control the behavior of the installed LXD server.
+For example, you can define a LXD user group to achieve a multi-user environment for LXD (see {ref}`projects-confine-users` for more information).
+
+See the [LXD snap page](https://snapcraft.io/lxd) for a list of available configuration options.
+
+To set any of these options, use the following command:
+
+    sudo snap set lxd <key>=<value>
+
+For example:
+
+    sudo snap set lxd daemon.user.group=lxd-users
+
+To see all configuration options that are set on the snap, use the following command:
+
+    sudo snap get lxd
+
+```{note}
+This command returns only configuration options that have been explicitly set.
+```
+
+See [Configure snaps](https://snapcraft.io/docs/configuration-in-snaps) in the snap documentation for more information about snap configuration options.
+
+## Start and stop the daemon
+
+To start and stop the LXD daemon, you can use the `start` and `stop` commands of the snap:
+
+    sudo snap stop lxd
+    sudo snap start lxd
+
+These commands are equivalent to running the corresponding `systemctl` commands:
+
+    sudo systemctl stop snap.lxd.daemon.service snap.lxd.daemon.unix.socket
+    sudo systemctl start snap.lxd.daemon.unix.socket; lxc list
+
+Stopping the daemon also stops all running instances.
+
+To restart the LXD daemon, use the `restart` command of the snap:
+
+    sudo snap restart lxd
```

Comment:
Due to this snap bug we should encourage this command:

https://bugs.launchpad.net/snapd/+bug/2028141